dm-tool: add page (#7710)
tldr-pages/tldr: pages/linux/dm-tool.md

# dm-tool

> > A tool to commnicate with the display manager. EG: lock currect session and show lightdm, list all open sessions, connect an existing x server to the display manager.

Reviewer: But the examples are in the examples below. :thinking: 
```suggestion
> A tool to communicate with the display manager.
```

> > More informatioin: https://www.mankier.com/1/dm-tool.

Reviewer 2: ```suggestion
> More information: <https://www.mankier.com/1/dm-tool>.
```

Reviewer 3: @godalming123, please prefer [manned.org](https://manned.org/dm-tool.1) when it's possible.

Author: Ok I won't necessarily use maned.org but will accept all of the other changes here on Monday night

Reviewer 2: Unless there is a valid reason, the manned page for this must be used. I see no difference between the current link and the one on manned.org

- Show the greeter while keeping current desktop session in ram and waiting to be restored upon autentication by logged in user:

`dm-tool switch-to-greeter`

- Lock the current session:

`dm-tool lock`

- Swicth user, showing a authentication prompt if required:

> `dm-tool switch-to-user {{username}} [{{session}}]`

Author: but don't we still need to have the `{{session}}` thing as the command doesn't work without it

> `dm-tool switch-to-user {{username}} [{{session}}]`

Reviewer: If the man page has `[session]` with square brackets is because it's optional, if it's required:
```suggestion
`dm-tool switch-to-user {{username}} {{session}}`
```

> `dm-tool swict-to-user {{username}} [{{session}}]`

Reviewer 3: ```suggestion
`dm-tool switch-to-user {{username}} [{{session}}]`
```

Author: Ok will do on Monday evening

> - Add a new X server instance:

Reviewer: Based on [this](https://www.opensourceagenda.com/projects/xweston) 
```suggestion
- Add a dynamic seat from within a running LightDM session:
```

> `dm-tool add-seat [{{name}}={{value}}]`

Reviewer: ```suggestion
`dm-tool add-seat {{xlocal}} {{name}}={{value}}`
```
